src/Datadog.Trace.ClrProfiler.Managed.Core/ClrNames.cs

```diff
@@ -38,5 +40,11 @@ internal static class ClrNames
         public const string GenericParameterTask = "System.Threading.Tasks.Task`1<T>";
         public const string ObjectTask = "System.Threading.Tasks.Task`1<System.Object>";
         public const string Int32Task = "System.Threading.Tasks.Task`1<System.Int32>";
+
+        public const string MsmqCursorHandle = "System.Messaging.Interop.CursorHandle";
+        public const string MsmqMessageQueueTransaction = "System.Messaging.MessageQueueTransaction";
+        public const string MsmqMessageQueueTransactionType = "System.Messaging.MessageQueueTransactionType";
+        public const string MsmqMessagePropertyFilter = "System.Messaging.MessagePropertyFilter";
+        public const string MsmqMessage = "System.Messaging.Message";
```

Comment:
As much as possible, we should put in ClrNames only base types that may be useful for other integrations. For the MSMQ ones, you should declare them in the same folder as the integration (for instance in `MsmqConstants`)